kano_updater/ui/views/finish.py

```diff
@@ -62,3 +63,6 @@ def _finish(self, window=None, event=None):
             os.system('systemctl poweroff')
         else:
             os.system('systemctl reboot')
+
+        time.sleep(10)
+        os._exit(0)
```

Comment:
I knew I did this for a reason... `sys.exit()` does not work, it's blindly refusing to terminate.
I could do something like `if in a sysroot then os._exit()` ...